sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -278,6 +278,8 @@ public long putLog(@NonNull Log log, @NonNull String group, @IntRange(from = Fla
             AppCenterLog.debug(LOG_TAG, "Storing a log to the Persistence database for log type " + log.getType() + " with flags=" + flags);
             String payload = getLogSerializer().serializeLog(log);
             ContentValues contentValues;
+
+            //noinspection CharsetObjectCanBeUsed
```

Comment:
Can we actually implement it?

Reply:
No, it requires API 19

Reply:
Add a comment like when we disable the warning for try with finally

Reply:
Added as suggestion